ksqldb-functional-tests/src/test/resources/query-validation-tests/fk-join.json

```diff
@@ -193,6 +327,25 @@
         "message": "Invalid join condition: table-table joins require to join on the primary key of the right input table. Got LEFT_TABLE.ID1 = RIGHT_TABLE.F2."
       }
     },
+    {
+      "name": "Should fail on right non-key attribute for outer-join (reverse join condition order) - feature flag enabled",
```

Comment:
Why do we need this test? FK joins aren't supported for outer joins at all.

Reply:
Ah. Good catch. Seems I did to much "blind" c&p :)